app/models/upvs/box.rb

```diff
@@ -38,4 +45,10 @@ def self.create_with_api_connection!(params)
   def sync
     Govbox::SyncBoxJob.perform_later(self)
   end
+
+  private
+
+  def validate_settings_obo
+    errors.add(:settings_obo, "OBO must be in UUID format") if settings_obo.present? && !settings_obo.match?(Utils::UUID_PATTERN)
```

Comment:
nitpick: Tu by som pouzil dva riadky

```suggestion
    return unless settings_obo.present?
    errors.add(:settings_obo, "OBO must be in UUID format") unless settings_obo.match?(Utils::UUID_PATTERN)
```